heareval/tasks/dcase2016_task2.py

```diff
@@ -67,6 +72,9 @@ def requires(self):
     }
 
     def get_split_metadata(self, split: str) -> pd.DataFrame:
+        # Since the valid is part of the train
```

Comment:
I don't understand what this does or why this works can you explain better? It's very confusing

Reply:
Actually the get split metadata was using the split to iterate over and generate. But since there is no requires task for the valid split, this was supposed to be sampled. It is clear in the new commits